internal/chrome_desktop.py

```diff
@@ -25,7 +25,8 @@
     '--disable-default-apps',
     '--disable-domain-reliability',
     '--safebrowsing-disable-auto-update',
-    '--disable-background-timer-throttling'
+    '--disable-background-timer-throttling',
+    '--no-sandbox'
```

Comment:
I'm not comfortable adding --no-sandbox to the code generally.  If it is necessary for Docker then we'll need to make it a config option.